bucket/btop.json

```
@@ -0,0 +1,31 @@
+{
+    "version": "1.0.2",
+    "description": "btop++ for windows",
+    "homepage": "https://github.com/aristocratos/btop4win",
+    "license": "Apache-2.0",
+    "notes": [
+        "Has no support for GPU monitoring, CPU temperature monitoring and does not have accurate CPU clock monitoring.",
+        "Does not require admin rights to run. (But it's still higly recommended to run as admin, otherwise some process information will be missing.)"
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/aristocratos/btop4win/releases/download/v1.0.2/btop4win-x64.zip",
+            "hash": "df0f7ef62dabe1d6776cba76d03e9922b440f55d210707cc3691b8abea5b3220",
+            "extract_dir": "btop4win"
+        }
+    },
```

Nitpick: move this out to architecture in case of future 32-bit and ARM versions.

```suggestion
            "hash": "df0f7ef62dabe1d6776cba76d03e9922b440f55d210707cc3691b8abea5b3220"
        }
    },
    "extract_dir": "btop4win",
```